source/Octopus.Tentacle.Contracts/StartScriptCommandV2.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace Octopus.Tentacle.Contracts
+{
+    public class StartScriptCommandV2
+    {
+        [JsonConstructor]
+        public StartScriptCommandV2(string scriptBody,
+            ScriptIsolationLevel isolation,
+            TimeSpan scriptIsolationMutexTimeout,
+            string? isolationMutexName,
+            string[] arguments,
+            string taskId,
+            ScriptTicket scriptTicket,
+            TimeSpan durationToWaitForScriptToFinish)
+        {
+            Arguments = arguments;
+            TaskId = taskId;
+            ScriptTicket = scriptTicket;
+            DurationToWaitForScriptToFinish = durationToWaitForScriptToFinish;
```

Comment:
This is a new property to optionally allow us to wait for a duration for the script to finish in the StartScript call.

This will allow us to optimise for short scripts e.g. StartScript(wait max 5 seconds) -> CompleteScript dropping hte need to poll with GetStatus